bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxBindingConstants.java

```diff
@@ -66,6 +85,22 @@ public class SolaxBindingConstants {
     public static final String TIMESTAMP = "last-update-time";
     public static final String RAW_DATA = "raw-data";
 
+    // Totals
+    public static final String POWER_USAGE = "power-usage";
+    public static final String TOTAL_ENERGY = "total-energy";
+    public static final String TOTAL_BATTERY_DISCHARGE_ENERGY = "total-battery-discharge-energy";
+    public static final String TOTAL_BATTERY_CHARGE_ENERGY = "total-battery-charge-energy";
+    public static final String TOTAL_PV_ENERGY = "total-pv-energy";
+    public static final String TOTAL_FEED_IN_ENERGY = "total-feed-in-energy";
+    public static final String TOTAL_CONSUMPTION = "total-consumption";
+
+    // Today totals
+    public static final String TODAY_ENERGY = "today-energy";
+    public static final String TODAY_BATTERY_DISCHARGE_ENERGY = "today-battery-discharge-energy";
+    public static final String TODAY_BATTERY_CHARGE_ENERGY = "today9082-battery-charge-energy";
```

Comment:
Is this number intended? I see only `today-battery-charge-energy` in localConnectInverter.xml.

Reply:
Hm. Ugly typo/copy-paste issue? Thanks for noticing it!